pkg/watchdogd/watchdogd.go

```diff
@@ -71,153 +94,266 @@ func MonitorOops() error {
 	return nil
 }
 
-// Run runs the watchdog on the current goroutine. The USR1, USR2, STOP and
-// CONT signals are used to control, so consider using a dedicated process.
-// Consider using the watchdogd command in u-root. Cancelling the context will
-// leave with the armed/disarmed state as is.
-func Run(ctx context.Context, opts *DaemonOpts) error {
-	defer log.Println("watchdogd: Daemon quit")
-
-	signals := make(chan os.Signal, 5)
-	signal.Notify(signals, unix.SIGUSR1, unix.SIGUSR2)
-	defer signal.Stop(signals)
-
-	for {
-		wd, err := watchdog.Open(opts.Dev)
+// startServing enters a loop of accepting and processing next incoming watchdogd operation call.
+func startServing(l *net.UnixListener) {
+	log.Println("start serving, start for loop")
```

Comment:
Prefix this print with `watchdogd: `

Reply:
Done.

moved it to a helper function `printWatchdogdln` for better consistency control.